packages/unplugin/src/getReleaseName.ts

```diff
@@ -0,0 +1,38 @@
+import * as child_process from "child_process";
+import * as fs from "fs";
+import * as path from "path";
+
+function isGit(dir: string) {
+  return fs.existsSync(path.join(dir, ".git"));
+}
+
+function getBranchHead() {
+  return child_process.execSync("git rev-parse HEAD").toString().trim();
+}
+
+export function getReleaseName(releaseName?: string): string {
+  if (releaseName) {
+    return releaseName;
+  }
+
+  const ENV_VARS = [
+    "SENTRY_RELEASE",
+    "SOURCE_VERSION", // Heroku #1 https://devcenter.heroku.com/changelog-items/630
+    "HEROKU_SLUG_COMMIT", // Heroku #2: https://docs.sentry.io/product/integrations/deployment/heroku/#configure-releases
+    "CODEBUILD_RESOLVED_SOURCE_VERSION", // AWS CodeBuild: https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-env-vars.html
+    "CIRCLE_SHA1", // CircleCI: https://circleci.com/docs/2.0/env-vars/
+    "VERCEL_GIT_COMMIT_SHA", // Vercel docs: https://vercel.com/docs/concepts/projects/environment-variables#system-environment-variables
```

Comment:
Does the order matter here? If yes, is the order correct?
(Looking at it again, I think it doesn't but I'm not sure about Heroku...)

Reply:
Great question! It does. This is why `SENTRY_RELEASE` is on the top of the list, everything else is carrying equal weight. I was thinking to extract `SENTRY_RELEASE` from the rest of the pack but found it to be overkill

Reply:
Thanks for explaining. Makes sense! Then, let's make sure to add a comment that `SENTRY_RELEASE` should be the first item in the list.